thrift/src/main/thrift/contentatom.thrift

```diff
@@ -54,6 +54,10 @@ struct ContentChangeDetails {
 
 struct Flags {
   1: optional bool suppressFurniture
+  2: optional bool sensitive
+  3: optional bool legallySensitive
+  4: optional bool suppressRelatedContent
+  5: optional bool blockAds
```

Comment:
@susiecoleman @clloyd I though we agreed to have only `legallySensitive` for the time being?